### ISSUE TITLE
Type Error in Tabulate

### ISSUE THREAD

**Issue by butellyn** [PennBBL/fw-heudiconv#76]
**Describe the issue**
I tried to tabulate a project on Flywheel (bbl/AGGY_808689), and ran into a Type Error: "'<' not supported between instances of 'str' and 'datetime.datetime'"

**Were you running `fw-heudiconv` locally or from Flywheel's GUI?**
- Locally

**If you ran it locally, please describe your setup:**
 - OS: MacOS Mojave

 - Version: 0.1.1

**Please paste any relevant output below**
```
Traceback (most recent call last):
  File "/Users/butellyn/miniconda3/envs/flywheel/bin/fw-heudiconv-tabulate", line 8, in <module>
    sys.exit(main())
  File "/Users/butellyn/miniconda3/envs/flywheel/lib/python3.7/site-packages/fw_heudiconv/cli/tabulate.py", line 143, in main
    unique=args.unique)
  File "/Users/butellyn/miniconda3/envs/flywheel/lib/python3.7/site-packages/fw_heudiconv/cli/tabulate.py", line 44, in tabulate_bids
    seq_infos = get_seq_info(client, project_label, sessions)
  File "/Users/butellyn/miniconda3/envs/flywheel/lib/python3.7/site-packages/fw_heudiconv/query.py", line 173, in get_seq_info
    for key, val in session_to_seq_info(client, session, context).items():
  File "/Users/butellyn/miniconda3/envs/flywheel/lib/python3.7/site-packages/fw_heudiconv/query.py", line 107, in session_to_seq_info
    sorted_acquisitions = sorted(acquisitions, key=lambda x: x.timestamp or '')
TypeError: '<' not supported between instances of 'str' and 'datetime.datetime'
```

**Comment by butellyn** [comment]
Hm, didn't mean to add the Heuristic label.

**Comment by TinasheMTapera** [comment]
Could you please update your fw-heudiconv version, this error was fixed a while back 9eae8c1 

`pip install --upgrade fw-heudiconv`
